app/LSP/Handler/Hover.hs

```diff
@@ -108,30 +108,30 @@ freeVarToHoverMap loc ty = mkHoverMap loc msg
                     , "- Type `" <> ppPrint ty <> "`"
                     ]
 
-xtorToHoverMap :: Loc -> PrdCnsRep pc -> TST.Typ pol -> CST.NominalStructural -> HoverMap
+xtorToHoverMap :: Loc -> PrdCnsRep pc -> TST.Typ pol -> RST.NominalStructural -> HoverMap
 xtorToHoverMap loc pc ty ns =  mkHoverMap loc msg
   where
     msg :: Text
     msg = case pc of
-      PrdRep -> T.unlines [ "#### " <> ppPrint ns <> " constructor"
+      PrdRep -> T.unlines [ "#### " <> ppPrint (show ns) <> " constructor"
```

Comment:
Anstatt die `Show` Instanz zu verwenden kannst du in `Pretty` eine Instanz `PrettyAnn RST.NominalStructural` hinzufügen. Das müsste in dem gleichen Modul sein wo auch die Instanz `PrettyAnn CST.NominalStructural` definiert wird.